components/board/customlinux/board.go

```diff
@@ -81,23 +81,30 @@ func createNewBoard(
 	return &b, nil
 }
 
-func parseBoardConfig(filePath string) ([]genericlinux.PinDefinition, error) {
+//lint:ignore U1000 Ignore unused function temporarily
+func parsePinConfig(filePath string) ([]genericlinux.PinDefinition, error) {
```

Comment:
I'm pretty confident this function is used. Remember to remove the lint directive when you're ready.